inductiva/fluids/fluid_block/README.md

```diff
@@ -0,0 +1,43 @@
+# Fluid Block
+
+This scenario simulates the motion of a fluid in a cubic tank. 
+Fluid is initial set as a cube and then allowed to flow under the influence of gravity.
+The simulation is performed using the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
+method. Currently, this scenario runs with both [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/splishsplash.py) and [DualSPHysics](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/dualsphysics.py).
+
+This scenario allows users to play with the different properties of fluids, namely, the density and the kinematic viscosity of the fluid.
+Further, users can select the size of the initial fluid block, the initial position and the initial velocity. 
+After the scenario has been initialized, users are ready to launch the simulation. They can configure the simulation further with the following parameters: the particule radius (float), the simulation time (float), adaptive time step (float), the time step (float) and the output time step (float).
```

Comment:
Do you think it is necessary to put the type of each variable?

Reply:
Not sure... Just because in the DamBreak resolution is low, medium high. It doesn't hurt at the moment.